File: Examples/Features/RangeOperations/GetWorksheetUsedRange.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.RangeOperations
{
    public class GetWorksheetUsedRange : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            worksheet.Range["H6:M7"].Value = 1;
            worksheet.Range["J9:J10"].Merge();

            //set interior color for worksheet usedRange "H6:M10".
            var usedrange = worksheet.UsedRange;
            usedrange.Interior.Color = Color.LightBlue;
        }
    }
}

File: Examples.VB/Features/RangeOperations/GetWorksheetUsedRange.vb
Imports System.Drawing

Namespace Features.RangeOperations
    Public Class GetWorksheetUsedRange
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            worksheet.Range("H6:M7").Value = 1
            worksheet.Range("J9:J10").Merge()

            'set interior color for worksheet usedRange "H6:M10".
            Dim usedrange = worksheet.UsedRange
            usedrange.Interior.Color = Color.LightBlue
        End Sub
    End Class
End Namespace
